### PR TITLE
chore: remove OpenShiftService.triggerBuild

### PR DESCRIPTION
**Pull request by gastaldi**

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @gastaldi for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin.

**Comment by sbose78** [comment]
@gastaldi The change looks OK to me. Could we re-trigger the semaphore CI ?

**Comment by gastaldi** [comment]
I need to investigate why tests are failing before merging this PR

File: addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java [hunk 1]
```diff
@@ -165,14 +165,6 @@ public ConfigMap createJenkinsConfigMap(OsioProjectile projectile, GitRepository
         return cm;
     }
 
-    public void triggerBuild(OsioProjectile projectile) {
-        //TODO remove this call (the trigger build should already done by the webhook) and change the countdown latch to 1 in OsioStatusClientEndpoint
-        String namespace = tenant.getDefaultUserNamespace().getName();
-        openShiftService.triggerBuild(projectile.getOpenShiftProjectName(), namespace);
-
-        projectile.getEventConsumer().accept(new StatusMessageEvent(projectile.getId(), OPENSHIFT_PIPELINE));
```
Comment (jarifibrahim) [inline addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java]: @gastaldi If this is being removed, shouldn't the `OPENSHIFT_PIPELINE` event removed from 
https://github.com/fabric8-launcher/launcher-backend/blob/577011eaa5934b8897d5f7e722f37ff9f49ce393/addons/osio-addon/src/main/java/io/fabric8/launcher/osio/web/endpoints/OsioImportEndpoint.java#L63

and 

https://github.com/fabric8-launcher/launcher-backend/blob/577011eaa5934b8897d5f7e722f37ff9f49ce393/addons/osio-addon/src/main/java/io/fabric8/launcher/osio/web/endpoints/OsioLaunchEndpoint.java#L64

Comment (gastaldi) [reply addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java]: @jarifibrahim the `OPENSHIFT_PIPELINE` event is sent when the createJenkinsConfigMap method is called

Comment (jarifibrahim) [reply addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java]: Oh yes. My bad. Thanks @gastaldi .

On a side note - why are we sending two events with the same name?

Comment (gastaldi) [reply addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java]: I wonder about that too. Probably that was before the StatusEventKind was turned into an interface.

Comment (gastaldi) [reply addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/OpenShiftSteps.java]: I wonder about that too. Probably that was before the StatusEventKind was turned into an interface.

**Comment by gastaldi** [comment]
@sbose78 PR passed. Will merge it first thing Monday

**Comment by sbose78** [comment]
Thanks George.

**Comment by gastaldi** [comment]
@sbose78 PR merged

**Pull request merged**